packages/core/src/components/discord-embed/DiscordEmbed.ts

```diff
@@ -423,7 +423,7 @@ export class DiscordEmbed extends LitElement implements DiscordEmbedProps, Light
 
 		return el.map((wordOrHtmlTemplate) => {
 			if (typeof wordOrHtmlTemplate === 'string') {
-				return html`<span>${wordOrHtmlTemplate}</span>`;
+				return html`<div><span>${wordOrHtmlTemplate}</span></div>`;
```

Comment:
```suggestion
				return html`<div>${wordOrHtmlTemplate}</div>`;
```

Oh I guess I misunderstood the contribution a bit. You can drop the span entirely then I think, at least that's what I conclude from my own testing.